pyproject.toml

```diff
@@ -196,7 +197,7 @@ oidc = ["authlib"]
 systemd = ["systemd-python"]
 url_preview = ["lxml"]
 sentry = ["sentry-sdk"]
-opentracing = ["jaeger-client", "opentracing"]
```

Comment:
Is `jaeger-client`/`jaeger_client` needed outside of `opentracing`, I see a bunch of `mypy` errors about it, https://github.com/matrix-org/synapse/runs/7639488053?check_suite_focus=true

Reply:
Not seeing this problem anymore. Don't know why it disappeared.